src/ovh-ui-angular/controllers/header-tabs.controller.js

```diff
@@ -0,0 +1,13 @@
+export default class {
+    constructor () {
+        "ngInject";
+
+        this.disableTab3 = true;
```

Comment:
Why are those attributes called this way ? It's hard to understand, maybe I'm missing something

Also, what about the other tabs ? 1 and 2 ?

Reply:
This is required to show the dynamic binding capacity of the component. It's not required for tab 1 and 2 as they are always enabled.